README.md

```diff
@@ -1,6 +1,6 @@
 # SQL style guide
 
-**[Read the guide](http://www.sqlstyle.guide)**
+**[Read the guide](includes/sqlstyle.guide.md)**
 
 ---
```

Comment:
please also change the "The markdown source for the guide can be found in _includes/sqlstyle.guide.md" link to refer to this repo